src/components/common/image-resizer.svelte

```diff
@@ -56,12 +57,17 @@
         {#if error != null}
           {error}
         {/if}
-        <Button isDanger on:click={() => isOpen = false}>cancel</Button>
+        <Button isDanger on:click={() => isOpen = false} disabled={uploading}>cancel</Button>
         <Button
           isFilled
-          on:click={() => dispatch('image-cropped', pixels)}
+          on:click={() => {
+            uploading = true;
+            dispatch('uploading', uploading);
+            dispatch('image-cropped', pixels);
+          }}
+          disabled={uploading}
         >
-          upload
+          { uploading ? 'uploading...' : 'upload' }
```

Comment:
As for these braces, I wasn't putting spaces around them :) Only objects and code blocks, not Svelte JS embeddings